runtime/testy/runtime.py

```diff
@@ -95,6 +95,8 @@ def nonTestModePrint(*args):
         print(newBadThing.event)
         nonTestModePrint(newBadThing.data)
         if newBadThing.event in restartEvents:
+          if newBadThing.event in studentErrorEvents:
+            stateQueue.put([SM_COMMANDS.SEND_CONSOLE, [newBadThing.stackTrace]])
```

Comment:
I think the newBadThing.event contains the enum for the actual bad event that we are checking in this case.

Also, the send console command expects a string rather than an object and we are directly setting a field in the proto to the argument used. We aren't printing it so I don't think the str method of the object would be called.